src/pages/BorrowerProfile/fundedBorrowerProfile.vue

```diff
@@ -28,7 +28,7 @@
 				<!-- Borrower location -->
 				<div>
 					<span class="country-image">
-						<kv-flag :country="`${ loan.geocode.country.isoCode }`" />
+						<kv-flag :country="`${ loan.geocode.country.isoCode }`" :custom-height="14" />
```

Comment:
`customHeight` is a backwards compatibility feature. The flags are actually square with white bars on top and bottom, but we cut off the white pixels at the bottom of the flag historically.